input/standardized-verification/notebook.ipynb

```diff
@@ -0,0 +1,1518 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Necessary Module and Set Path "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as lite\n",
+    "import os\n",
+    "from IPython.display import Image\n",
+    "import sys\n",
+    "import numpy as np\n",
+    "import collections\n",
+    "sys.path.insert(0, '../../scripts/')\n",
+    "jupyter_directory = os.getcwd()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import analysis as an"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Must use Cycamore in jbae11 cycamore branch \"standardized_verif\"\n",
```

Comment:
This is quite problematic. It undermines the publication.

Reply:
This problem has been solved by deciding to merge the changes into the cyclus/cycamore develop branch as an [optional feature](https://github.com/cyclus/cycamore/pull/489)